README.md

```diff
@@ -1,14 +1,15 @@
-# HarbourBridge: Turnkey Postgres-to-Spanner Evaluation
+# HarbourBridge: Turnkey Postgres-to-Spanner/MySQL-to-Spanner Evaluation
+
 [![cloudspannerecosystem](https://circleci.com/gh/cloudspannerecosystem/harbourbridge.svg?style=svg)](https://circleci.com/gh/cloudspannerecosystem/harbourbridge)
 
 HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation,
-using data from an existing PostgreSQL database. The tool ingests pg_dump
-output, automatically builds a Spanner schema, and creates a new Spanner
-database populated with data from pg_dump.
+using data from an existing PostgreSQL/MySQL database. The tool ingests from
+database/dump file output, automatically builds a Spanner schema, and creates
+a new Spanner database populated with data from the selected approach.
 
 HarbourBridge is designed to simplify Spanner evaluation, and in particular to
-bootstrap the process by getting moderate-size PostgreSQL datasets into Spanner
-(up to a few GB). Many features of PostgreSQL, especially those that don't map
+bootstrap the process by getting moderate-size PostgreSQL/MySQL datasets into Spanner
+(up to a few GB). Many features of PostgreSQL/MySQL, especially those that don't map
```

Comment:
Given the experiments you have run and also ones we've run, should we consider relaxing this a bit.
Maybe say: "up to a few tens of GB"?

Reply:
Fair enough.